### DISCUSSION TITLE
Potential Xiaogui TZC4 scale variant - is it compatible?

### DISCUSSION THREAD

**Discussion by przemhb** [dckiller51/bodymiscale#42] [Q&A] [answered]
Hi,

I've noticed my scale was detected as a Xiaogui TZC4, whereas it unbranded product with a bit different look, inverted LCD display and temperature measurement (I doubt if broadcasted). It works with "OKOK" Android app.

I'm not sure if it is compatible with the TZC4 as I get no readouts from the scale. All the values are "unknown" - impedance, weight, RSSI. 
Example packet send by the scale for 81,30 kg:

0x10FFC02A**1FC2**137D544E253F151B4329AF

Impedance is the word preceding the weight.

Is my scale compatible with TZC4 or MiScale2 and I miss something in its configuration or is it incompatible?
Could you shed some light onto this?

regards
Przemysław

**Comment by dckiller51** [comment, marked as answer]
The component does not retrieve the data but reads it only. To retrieve the data you have to look at esphome or ble monitor.
